src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,195 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
```

Comment:
class should not be a static
```suggestion
    private class Node<T> {
```

Reply:
OK

Reply:
But in LinkedList.java this class is static...